electrum/wallet.py

```diff
@@ -1961,6 +1973,17 @@ def fee_estimator(size: Union[int, float, Decimal]) -> int:
             # note: Actually, it might be the case that not all UTXOs from the wallet are
             #       being spent if the user manually selected UTXOs.
             sendable = sum(map(lambda c: c.value_sats(), coins))
+            if self.should_keep_reserve_utxo(sendable, is_channel_opening):
```

Comment:
what about non-"spend max" branch?
the user might try to send an amount very close to the available balance, where the only remaining utxo after the tx is the change output, which is below the reserve

Reply:
indeed. should_keep_reserve_utxo is now called in both branches of the if